File: Examples/PackageDiagramDiff/Python/After/Core/account.py
from money import Money


class Account:
    """A bank account holding a balance. Part of the ``Core`` module."""

    def __init__(self, id: str, balance: Money):
        self.id: str = id
        self.balance: Money = balance

    def deposit(self, amount: Money) -> None:
        self.balance = self.balance.adding(amount)

File: Examples/PackageDiagramDiff/Java/After/Core/Account.java
/** A bank account holding a balance. Part of the `Core` module. */
final class Account {
    final String id;
    private Money balance;

    Account(String id, Money balance) {
        this.id = id;
        this.balance = balance;
    }

    void deposit(Money amount) {
        balance = balance.adding(amount);
    }
}
